assets/xml/overlays/ovl_En_Bom.xml

```diff
@@ -1,9 +1,20 @@
 <Root>
+    <!-- Assets for the Powder Keg -->
     <File Name="ovl_En_Bom" BaseAddress="0x80870DB0" RangeStart="0x2160" RangeEnd="0x3610">
-        <DList Name="ovl_En_Bom_DL_80872F98" Offset="0x21E8"/>
-        <DList Name="ovl_En_Bom_DL_80873020" Offset="0x2270"/>
-        <DList Name="ovl_En_Bom_DL_80873040" Offset="0x2290"/>
-        <DList Name="ovl_En_Bom_DL_80873CA0" Offset="0x2EF0"/>
-        <DList Name="ovl_En_Bom_DL_808742F8" Offset="0x3548"/>
+        <!-- Texture and DisplayLists for the Powder Keg's fuse -->
+        <Texture Name="gPowderKegFuseTex" OutName="powder_keg_fuse" Format="i8" Width="2" Height="2" Offset="0x21E0"/>
+        <DList Name="gPowderKegFuseMaterialDL" Offset="0x21E8"/>
+        <DList Name="gPowderKegFuseModel1DL" Offset="0x2270"/>
+        <DList Name="gPowderKegFuseModel2DL" Offset="0x2290"/>
+
+        <!-- Textures and DisplayList for the Powder Keg -->
+        <Texture Name="gPowderKegTLUT" OutName="powder_keg_tlut" Format="rgba16" Width="16" Height="16" Offset="0x22B0"/>
+        <Texture Name="gPowderKegTex" OutName="powder_keg" Format="ci8" Width="32" Height="32" Offset="0x24B0"/>
+        <DList Name="gPowderKegDL" Offset="0x2EF0"/>
```

Comment:
I think I used `gPowderKegBarrelDL` instead of just a generic `gPowderKegDL` here, but it would be good double check if they're the same or different

Reply:
GI version:
![image](https://user-images.githubusercontent.com/12245827/180047280-9222f3af-db7c-4774-99bd-23b857aa80e5.png)

En_Bom version:
![image](https://user-images.githubusercontent.com/12245827/180047248-6bd16e05-e8d8-4ef0-a3b1-69ac1d66f364.png)

The scale is *way* different, and the asset itself is subtly different, but yeah, it's the same thing. I'll put "barrel" in mine too.